test/foundry/new/FuzzEngine.t.sol

```diff
@@ -493,6 +492,9 @@ contract FuzzEngineTest is FuzzEngine {
                 orders[0].toBasicOrderParameters(orders[0].getBasicOrderType())
             );
 
+        context.registerCheck(this.check_orderFulfilled.selector);
+        context.registerCheck(this.check_orderStatusFullyFilled.selector);
```

Comment:
`registerCheck` returns the `TestContext`, so these are chainable if you want, e.g.:

```
context.registerCheck(this.check_orderFulfilled.selector).registerCheck(this.check_orderStatusFullyFilled.selector);
```

Reply:
I forgot that.  I just changed it and prettier makes it uglier.  imo it's more readable as-is, but I'm happy to chain if we want to adopt chain-when-possible or something like that as a convention.